iothub/service/samples/getting started/JobsSample/JobsSample.cs

```diff
@@ -46,31 +45,42 @@ public async Task RunSampleAsync()
             try
             {
                 Console.WriteLine($"Schedule twin job {jobId} for {DeviceId}...");
-                JobResponse createJobResponse = await _jobClient
+
+                var scheduledTwinUpdate = new ScheduledTwinUpdate
+                {
+                    QueryCondition = query,
+                    Twin = twin,
+                    StartOnUtc = DateTime.UtcNow
+                };
+
+                var jobOptions = new ScheduledJobsOptions
+                {
+                    JobId = jobId,
+                    MaxExecutionTime = TimeSpan.FromMinutes(2)
+                };
+
+
+                ScheduledJob createJobResponse = await _jobClient.ScheduledJobs
                     .ScheduleTwinUpdateAsync(
-                        jobId,
-                        query,
-                        twin,
-                        DateTime.UtcNow,
-                        (long)TimeSpan.FromMinutes(2).TotalSeconds);
+                        scheduledTwinUpdate,
```

Comment:
@timtay-microsoft Seems that the items in `ScheduledTwinUpdate` should just be parameters, if they are required, and in `ScheduledJobOptions` if they are not.

Thoughts?

Reply:
We could make that change, but the parameter list would be rather large. 3 mandatory parameters, one options bundle, and one cancellation token.

Reply:
5 doesn't seem too terrible. I don't recall what the Azure SDK team recommends.